src/app/src/test/kotlin/org/imperial/mrc/hint/unit/logging/LoggingTests.kt

```diff
@@ -0,0 +1,60 @@
+package org.imperial.mrc.hint.unit.logging
+
+import com.nhaarman.mockito_kotlin.mock
+import com.nhaarman.mockito_kotlin.verify
+import net.logstash.logback.argument.StructuredArguments.kv
+import org.imperial.mrc.hint.logging.*
+import org.junit.jupiter.api.Test
+
+class LoggingTests
```

Comment:
Could we call this class and file GenericLoggerImplTests if that's what it's testing? That would follow our standard pattern.